app/src/main/res/values/styles.xml

```diff
@@ -269,4 +270,12 @@
         <item name="actionModeCloseDrawable">@drawable/ic_close_white_24dp</item>
     </style>
 
+    <style name="ChatRooms.BottomSheetDialog" parent="Theme.Design.Light.BottomSheetDialog">
+        <item name="bottomSheetStyle">@style/BottomSheetStyle</item>
+    </style>
+
+    <style name="BottomSheetStyle" parent="Widget.Design.BottomSheet.Modal">
+        <item name="behavior_peekHeight">512dp</item>
```

Comment:
Where did you find this value? From an official reference?

Reply:
@filipedelimabrito I found it in bug tracker list of google and others had this problem so one of them suggested this solution. Is there anything wrong with this?

Reply:
I was just asking. Can you send the bug tracker link for this? So I can (and others too) take a look on it too. Thank you.

Reply:
@filipedelimabrito here is the link https://issuetracker.google.com/issues/37132390#c6

Reply:
@filipedelimabrito No I have tested on my phone and emulator seems its working fine.

Reply:
@Cool-fire Make sure it works as well on small-screen devices. I would also say it would be better if you can make the parent layout of every bottom sheet layout we have to have a height of `wrap_content` instead of `match_parent`, and add a ScrollView to every layout after the bottom sheet title divider, since at certain points, one may need to scroll if the list is very long on the device screen.

Reply:
@Cool-fire Make sure it works as well on small-screen devices. I would also say it would be better if you can make the parent layout of every bottom sheet layout we have to have a height of `wrap_content` instead of `match_parent`, and add a `ScrollView` to every layout after the bottom sheet title divider, since at certain points, one may need to scroll if the list is very long on the device screen.